baseband/vlbi_base/tests/test_vlbi_base.py

```diff
@@ -425,34 +425,147 @@ def test_vlbi_file_base(self, tmpdir):
             assert fh.closed
             assert fh.fh_raw.closed
 
-    def test_streambase_squeeze(self):
-        # Tests stream base's sample shape concatenation routines.
-        smp_shp_cls = namedtuple('SampleShape',
-                                 'n1, n2, n3, n4, n5, n6, n7, n8')
-        sample_shape = smp_shp_cls(1, 17, 3, 2, 1, 5, 1, 1)
+    def make_streamwriter_with_shape(self, smp_shp_mkr, unsliced_shape,
```

Comment:
Why not just call the argument `sample_shape_maker`?

Reply:
To stop having to wrap so many lines.  I've expanded the variable as suggested, while shortening the function name.

Reply:
OK, sounds good.